etna/articles/blocks.py

```diff
@@ -200,6 +200,7 @@ class FeaturedCollectionBlock(SectionDepthAwareStructBlock):
     description = blocks.TextBlock(max_length=200)
     items = PageListBlock(
         "articles.ArticlePage",
+        "articles.RecordArticlePage",
```

Comment:
@jamesbiggs Can we just double-check that we're using `.specific` when pulling these values out for display (to get the actual `ArticlePage` or `RecordArticlePage`)? It only does that for you automatically when you specify a single option

Reply:
Hey Andy - looks like specific is being used already, here's an example of the element:
![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/1edfe621-f1a6-4ade-80d4-84b36bb4311e)

😄

Reply:
NOTE: Resolved on slack - added `.specific` to the template as it was coming up funny in preview mode 🙂